app.json

```diff
@@ -0,0 +1,30 @@
+{
+  "addons": ["heroku-postgresql:hobby-free"],
+  "buildpacks": [
+    { "url": "heroku/nodejs" },
+    { "url": "https://github.com/moneymeets/python-poetry-buildpack.git" },
+    { "url": "heroku/python" },
+    { "url": "heroku-community/cli" }
+  ],
+  "env": {
+    "ERR_SESSION_SECRET": {
+      "description": "A secret key for verifying the integrity of signed cookies.",
+      "generator": "secret"
+    }
+  },
+  "formation": {
+    "web": {
+      "quantity": 1,
+      "size": "standard-1x"
+    },
+    "worker": {
+      "quantity": 1,
+      "size": "standard-1x"
+    }
```

Comment:
same, remove this until we need it.

Reply:
I don't think we have a worker yet, right? Let's remove it from the formation for now.